src/Digdir.Domain.Dialogporten.GraphQL/Program.cs

```diff
@@ -97,6 +111,9 @@ static void BuildAndRun(string[] args)
 
     var app = builder.Build();
 
+    app.MapAspNetHealthChecks()
+        .MapControllers();
```

Comment:
Her skal vi ikke kalle MapControllers. Det er egentlig en "gammel" metode som ikke brukes lenger etter minimal apis kom på banen. Eneste grunnen for at vi gjør det i webapi er fordi vi må støtte JsonPatchDocument som kun støttes av controllers (eller i hvert fall gjorde det).